### ISSUE TITLE
Update R3F_LOG version to latest?

### ISSUE THREAD

**Issue by 1datatec** [A3Wasteland/ArmA3_Wasteland.Stratis#98]
Hey, i've done the update myself and was using latest r3f version 3.1 but now i saw that you are actively updating old r3f which is in the mission, so i reverted to your version. Any chance you update it to latest on your side? It's as simple as downloading and editing path in description, init and sell crate.
Advantages:
Ability to rise/lower objects, rotate objects, turn objects, move them closer/further with hotkeys.
Also it has a script to autoload objects in vehicles/crates which is useful to for example airdrop crate with baseparts inside.
Disadvantages:
None.

**Comment by AgentRev** [comment]
It's not simple. The R3F code has a ton of bugs and optimization issues, many of which are still present in v3. For instance, `surveiller_conditions_actions_menu.sqf` from v3 needs to be completely rewritten from scratch because it is absolute garbage which is very hard to customize. I have corrected a lot of issues over the years, and implemented new features such as heavy towing/lifting and paradropping. Porting all of this to v3 and fixing its issues could take 1 to 2 weeks of full-time work. I've actually considered doing this for years now (v3 was released in 2014...) but I always find other stuff that I consider more important to work on. It is possible I might work on it this summer, but don't hold your breath.
